install/kubernetes/helm/istio/charts/mixer/values.yaml

```diff
@@ -31,6 +31,12 @@ telemetry:
     # based on measurements 100ms p50 translates to p99 of under 1s. This is ok for telemetry which is inherently async.
     latencyThreshold: 100ms
 
+  gcpCredential:
+    enabled: false
+    secretName: gcp-creds
+    volumeName: gcp-credentials
+    volumeMountPath: /etc/gcp-credentials
```

Comment:
1. Please stop using /etc - /var or other dirs are commonly used for this. 
2. Again, we already have 100s useless options that confuse users. Is there any case where this default name will not work and someone will want to call it something else ? K8S has well defined paths where it loads the secret - why does mixer needs an extra layer of obfuscation ? 

And don't call it 'gcp' - I don't want to see 10 other options named by each platform. 
It's a dir where multiple secrets can be loaded - one may be called gcp and another one aws.

Reply:
Done.